docker/gcp_pudl_etl.sh

```diff
@@ -112,13 +114,14 @@ function zenodo_data_release() {
     IGNORE_REGEX=$3
     PUBLISH=$4
 
-    # Trigger the zenodo data release workflow using the GitHub API
-    curl -sS -X POST \
-        -H "Accept: application/vnd.github+json" \
-        -H "Authorization: Bearer ${PUDL_BOT_PAT}" \
-        https://api.github.com/repos/catalyst-cooperative/pudl/actions/workflows/zenodo-data-release.yml/dispatches \
-        -d @<(
-            cat <<JSON
+    set +x &&
```

Comment:
I forgot to use `set +x` here so I probably need to go generate a new `PUDL_BOT_PAT` since it showed up in the logging output.